src/Kafka/CreateKafkaInstance/components/BillingTiles.tsx

```diff
@@ -192,7 +194,8 @@ export const BillingTiles: VoidFunctionComponent<BillingTilesProps> = ({
 };
 
 const tiles: { [id in MarketPlace]: JSX.Element } = {
-  aws: <AwsIcon size="md" color="black" />,
-  azure: <MicrosoftIcon size="md" color="black" />,
+  aws: <AWSLogo />,
+  gcp: <GCPLogo />,
+  azure: <AzureLogo />,
   rhm: <RedhatIcon size="md" color="black" />,
```

Comment:
@jgiardino @riccardo-forina Do you guess know from where I can get image for Redhat logo?